darks_driftwind_crypt/developmental_testing.dsc

```diff
@@ -123,22 +123,22 @@ region_handler:
 
 fancy_teleport:
   type: task
-  definitions: location
+  definitions: location_name
   script:
   # TODO: This title insertion / fade depends on: - resource pack data and structure
     - title title:<&font[fade:black]><&chr[0004]><&chr[F801]><&chr[0004]> fade_in:5t stay:0s fade_out:1s
     - wait 5t
-    - teleport <[location]>
+    - teleport <location[<[location_name]>]>
```

Comment:
Can you re-verify that this doesn't work? Seems like a bug to report to upstream

Reply:
yes
https://paste.denizenscript.com/View/89969